test-harness/src/test/java/com/netflix/conductor/test/integration/grpc/GrpcEndToEndTest.java

```diff
@@ -13,12 +13,14 @@
 package com.netflix.conductor.test.integration.grpc;
 
 import org.junit.Before;
+import org.junit.Ignore;
 
 import com.netflix.conductor.client.grpc.EventClient;
 import com.netflix.conductor.client.grpc.MetadataClient;
 import com.netflix.conductor.client.grpc.TaskClient;
 import com.netflix.conductor.client.grpc.WorkflowClient;
 
+@Ignore
```

Comment:
> 2 tests which depends on the client changes are marked as @ignore as they need to be updated with new client changes. These can be re-enabled.

Do these need to be re-enabled in this PR? Or will that be done as part of client change PR?

Reply:
Yes re-enabled them now that client is fixed.